src/test/kotlin/eli/inevitable/KotlinCellFormatterAPITest.kt

```diff
@@ -0,0 +1,60 @@
+package eli.inevitable
+
+import com.thedeanda.lorem.LoremIpsum
+import eli.inevitable.enumerations.Align.CENTER
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class KotlinCellFormatterAPITest {
+    private val faker: LoremIpsum = LoremIpsum.getInstance()
+
+    @Test
+    fun `Example vararg builder output`() {
```

Comment:
Spot checking these since there's no assertions I assume?

Reply:
Nope, these are APITests that just test out various API call patterns. It helps me see how the API will look when it's being consumed. The purpose isn't to "validate" so much as to demonstrate how the pieces are called and would consequently display to the screen.